### DISCUSSION TITLE
restore is cancelled

### DISCUSSION THREAD

**Discussion by wuchencm** [apache/doris#10976] [RFC]
[COMMON_ERROR, msg: Could not set meta version to 99 since it is lower than minimum required version 100] 
目标集群  To doris cluster：doris version 1.1.0 
源集群 from doris cluster： doris version 0.14.0

`RESTORE SNAPSHOT fulink_db.snapshot_20220718 
from dorisBackup_hdfs
PROPERTIES(
"backup_timestamp" = "2022-07-18-08-42-28"
)`

**Comment by hf200012** [comment]
不能跨版本恢复数据

**Comment by wuchencm** [comment]
我从0.14恢复到0.15的集群是可以的，那请问有什么好的办法能将数据恢复到1.1的集群里吗？ 在0.14集群上升级1.1 风险有点大

**Comment by kpfly** [comment]
别跨越版本，从0.14 ->0.15 -> 1.0 -> 1.1

**Reply by wuchencm** [reply]
感谢
